staging/src/k8s.io/apimachinery/pkg/runtime/interfaces.go

```diff
@@ -210,6 +210,18 @@ type ObjectCreater interface {
 	New(kind schema.GroupVersionKind) (out Object, err error)
 }
 
+// ResourceMapper provides information about equivalent resources,
+// and the kind associated with a given resource
```

Comment:
missing the role of subresources. Do you identify gvr+subresource with other gvr+subresource values?

Reply:
documented meaning of subresource in each function